spec/controllers/chapters_controller_spec.rb

```diff
@@ -962,27 +960,58 @@
 
           old_updated_at = work.updated_at
 
-          delete :destroy, params: { work_id: work.id, id: @chapter2.id }
+          delete :destroy, params: { work_id: work.id, id: chapter2.id }
           expect(assigns[:work].updated_at).not_to eq(old_updated_at)
         end
 
         it "gives a notice that the chapter was deleted and redirects to work" do
-          delete :destroy, params: { work_id: work.id, id: @chapter2.id }
+          delete :destroy, params: { work_id: work.id, id: chapter2.id }
           it_redirects_to_with_notice(work, "The chapter was successfully deleted.")
         end
 
         it "gives a notice that the draft chapter was deleted if the chapter was a draft and redirects to work" do
-          @chapter2.posted = false
-          @chapter2.save
-          delete :destroy, params: { work_id: work.id, id: @chapter2.id }
+          chapter2.posted = false
+          chapter2.save
+          delete :destroy, params: { work_id: work.id, id: chapter2.id }
           it_redirects_to_with_notice(work, "The chapter draft was successfully deleted.")
         end
 
         it "errors and redirects to work when chapter is not deleted" do
           allow_any_instance_of(Chapter).to receive(:destroy).and_return(false)
-          delete :destroy, params: { work_id: work.id, id: @chapter2.id }
+          delete :destroy, params: { work_id: work.id, id: chapter2.id }
           it_redirects_to_with_error(work, "Something went wrong. Please try again.")
         end
+
+        it "does not reorder chapters when deleting the last chapter" do
```

Comment:
For reference: this first test and the last test (`"reorders chapters properly when deleting a mid-work chapter"`) pass on both this branch and master. The two in the middle around deleting the first chapter pass on this branch but fail on master.